### DISCUSSION TITLE
百度网盘超级会员，Alist的v3版本要怎么设置，才能通过RaiDrive挂载到本地看视频

### DISCUSSION THREAD

**Discussion by lklk1904** [AlistGo/alist#2681] [Q&A]
![image](https://user-images.githubusercontent.com/91129125/206909295-5cabfd6e-9e4d-45db-991b-246cbe00b03e.png)

**Comment by rakaki** [comment]
webdav策略要选成 本机代理；或者webdav策略不变 下载接口选 crack。你没好好看文档吧

**Reply by lklk1904** [reply]
大佬，你说的是V3版本吗？V3版本的下载接口选项是中文了，不是英文crack了。我有尝试如下操作：

![001](https://user-images.githubusercontent.com/91129125/206978207-45a42e15-7b5f-49a5-8331-95a3e64c3b6c.jpg)

![002](https://user-images.githubusercontent.com/91129125/206978230-8d161825-b8d4-4df2-a540-40c4e074fbd7.jpg)

![003](https://user-images.githubusercontent.com/91129125/206978258-8a02621d-e61a-4d6a-b984-bab18dcc5397.jpg)

![004](https://user-images.githubusercontent.com/91129125/206978278-87fe9bf7-4327-4561-abfd-9a1e0804ca95.jpg)

![005](https://user-images.githubusercontent.com/91129125/206978297-ca3418c8-52c5-41db-9363-9cd8a2827db3.jpg)
